README

```diff
@@ -48,14 +48,38 @@ infrastructure.
    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ----
 
-=== Building
-After downloading and unpacking the package tarball, you build it as
-follows.
+=== Building on POSIX
+Either clone from Git or download and unpackage the tarball, then make 
+sure you have the pre-requisites installed and build following the 
+steps below. 
 
+Please make sure to have recent versions of the following packages 
+installed on your system. 
+
+  autoconf automake libtool libssl pkg-config check libpcsc gengetopt 
+  help2man
+
+Help2man is used to generate the manpages.  
+Gengetopt version 2.22.6 or later is needed for command line parameter 
+handling. The 
+link:https://github.com/Yubico/yubico-piv-tool/tree/master/vagrant/development[Vagrant
+VM] has all these dependencies preinstalled.
+
+  $ autoreconf --install
```

Comment:
`autoreconf` should only be required if using a git checkout.  Not sure if that's worth pointing out.

Reply:
Sadly, this was not the case. Downloaded the previous release tarball and 'autoreconf' was required. Maybe this is something that has broken in our tarball-build pipelines?

Reply:
No, we probably never shipped the 'dist' version then.  It's safe to ignore this comment, then :)